tgui/packages/tgui/interfaces/GeneTek.js

```diff
@@ -81,7 +84,7 @@ export const GeneTek = (props, context) => {
                 Record
               </Tabs.Tab>
             )}
-            {!!haveSubject && (
+            {!!subject && (
```

Comment:
`subject` is either null or an object, so can just do (probably):
```suggestion
            {subject && (
```